test/presets/bem.js

```diff
@@ -151,6 +151,43 @@ doAsync(function() {
 
 // Comments with url can have maximum line length > 120 ................... https://github.com/yandex/codestyle/blob/master/js.md
 
+// `i-bem` should be named BEM
+// `i-bem__dom` should be named BEMDOM
+modules.define('something-truly', ['i-bem__dom', 'something-else'], function(provide, BEMDOM, SomethingElse) {
+
+// Should use `this.name` if possible for BEMDOM.decl
+// Should have @lends jsdoc tag
```

Comment:
I'm not sure, but seems that `@lends` may be only used with `@class` tag declared. In other case it won't work properly